.gitignore

```diff
@@ -5,3 +5,4 @@ sublime/User/*/
 *.last-run
 Package Control.ca-list
 *ca-bundle
+sublime/User/oscrypto-ca-bundle.crt
```

Comment:
Would you mind to explain the nature of this certificate file? I can't get it by myself :)